tools/datafix/delete_invalid.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+"""Utility to delete invalid bugs that match specific criteria.
+
+See https://github.com/google/osv.dev/issues/1098 for additional context.
+"""
+
+from google.cloud import datastore
+
+import argparse
+
+
+def main() -> None:
+  parser = argparse.ArgumentParser(
+      description="Delete bugs that are invalid and match specific criteria.")
+  parser.add_argument(
+      "--dry-run",
+      action=argparse.BooleanOptionalAction,
+      dest="dryrun",
+      default=True,
+      help="Abort before making changes")
+  parser.add_argument(
+      "--source_id_prefix",
+      action="store",
+      dest="source_id_prefix",
+      default="",
+      help="The prefix of source_id records to delete")
+  parser.add_argument(
+      "--project",
+      action="store",
+      dest="project",
+      default="oss-vdb-test",
+      help="GCP project to operate on")
+  args = parser.parse_args()
+
+  client = datastore.Client(project=args.project)
+
+  query = client.query(kind="Bug", filters=(("status", "=", 2),))
```

Comment:
nit: to make this slightly faster, you could also filter on source = ghsa.

I think we have the indexes available to do that.

Reply:
Thanks! Addressed in [dd0fe10](https://github.com/google/osv.dev/pull/1116/commits/dd0fe10d43562d4ab6cfb1f8ef97bc487a1f0d75)

```
$ ./tools/datafix/delete_invalid.py --project=oss-vdb --source_id_prefix="ghsa:advisories/unreviewed"
Running query [('status', '=', 2), ('source', '=', 'ghsa')] on Bug (in oss-vdb)...
Retrieved 5241 bugs to examine for deletion
There are 5075 bugs to delete...
Traceback (most recent call last):
  File "/usr/local/google/home/apollock/gosst/osv/osv.dev/./tools/datafix/delete_invalid.py", line 72, in <module>
    main()
  File "/usr/local/google/home/apollock/gosst/osv/osv.dev/./tools/datafix/delete_invalid.py", line 66, in main
    raise Exception("Dry run mode. Preventing transaction from commiting")  # pylint: disable=broad-exception-raised
Exception: Dry run mode. Preventing transaction from commiting
```